### ISSUE TITLE
Get monitors and adapters

### ISSUE THREAD

**Issue by Unweirdly** [Unweirdly/FirstDraft#6]

**Comment by Unweirdly** [comment]
@jonitaylor513 Do you remember what you will need for this. Also we need to get a desk up stairs to work from.

**Comment by jonitaylor513** [comment]
Yes, I need an DVG to HDMI adaptor. The Wal-Mart I went to that day did not have one.

**Comment by jonitaylor513** [comment]
https://www.amazon.com/AmazonBasics-HDMI-Output-Adapter-Cable/dp/B014I8UL8U/ref=sr_1_1_sspa?s=electronics&ie=UTF8&qid=1532905079&sr=1-1-spons&keywords=dvi+to+hdmi&smid=ATVPDKIKX0DER&psc=1
